metadata/location/utah_address_points

```diff
@@ -0,0 +1,100 @@
+# Title
+
+Utah Address Points
+
+## ID
+
+4373453d-9e20-44da-9c6e-1f0b1ea653af
+
+## Brief Summary
+
+Contains US Postal Service address points for all twenty-nine counties in Utah.
+
+## Summary
+
+An address point represents a geographic location that has been assigned a US Postal Service (USPS) address by the local address authority (i.e., county or municipality) but does not necessarily receive mail. This data layer contains address points for all twenty-nine counties in Utah.
```

Comment:
In a summary, I have found it to be good practice to specify the extent of a given layer, but since the coverage is already mentioned in the Brief Summary, I'll go ahead and take it out.